src/components/search/Filter/facets.module.scss

```
@@ -10,17 +10,17 @@
   --fs-facets-title-margin-bottom                        : var(--fs-spacing-0);
 
   // Accordion
-  --fs-facets-accordion-notebook-border-width            : 1px;
-  --fs-facets-accordion-notebook-border-color            : var(--fs-border-color-light);
-  --fs-facets-accordion-notebook-border-radius           : var(--fs-border-radius);
+  --fs-facets-accordion-border-width-notebook            : 1px;
```

```suggestion
  --fs-facets-accordion-border-width-notebook            : var(--fs-border-width);
```

😅 Can you update all the occurrences? thanks

---

done [here](https://github.com/vtex-sites/nextjs.store/pull/150/commits/7dece2f2f7edf0e096ec3edeb118eca4d626e00e).